e2e/cypress/integration/onboarding/use_team_invite_link_to_sign_up_spec.js

```diff
@@ -0,0 +1,133 @@
+
+describe('Onboarding', () => {
+    let testTeam;
+    const randomId = getRandomId();
+    const username = `user${randomId}`;
+    const email = `user${randomId}@sample.mattermost.com`;
+    const password = 'passwd';
+
+    const baseUrl = Cypress.config('baseUrl');
+    const mailUrl = getEmailUrl(baseUrl);
+
+    before(() => {
+        // # Do email test if setup properly
+        cy.apiEmailTest();
+
+        // # Update config to require email verification
+        cy.apiUpdateConfig({
+            EmailSettings: {
+                RequireEmailVerification: true,
+            },
+        });
+
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+            cy.visit(`/${team.name}`);
+        });
+    });
+
+    it('MM-T398 Use team invite link to sign up using email and password', () => {
+        // # Open the 'Invite People' full screen modal and get the invite url
+        cy.get('.sidebar-header-dropdown__icon').click();
+        cy.get('#invitePeople').find('button').eq(0).click();
+        cy.get('.share-link-input').invoke('val').then((val) => {
+            const inviteLink = val;
+
+            // # Logout from admin account and visit the invite url
+            cy.apiLogout();
+            cy.visit(inviteLink);
+        });
+
+        // # Type email, username and password
+        cy.wait(TIMEOUTS.HALF_SEC);
+        cy.get('#email').type(email);
+
+        cy.wait(TIMEOUTS.HALF_SEC);
+        cy.get('#name').type(username);
+
+        cy.wait(TIMEOUTS.HALF_SEC);
+        cy.get('#password').type(password);
+
+        // # Attempt to create an account by clicking on the 'Create Account' buton
+        cy.get('#createAccountButton').click();
+
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // * Check that 'Mattermost: You are almost done' text should be visible when email hasn't been verified yet
+        cy.get('.signup-team__container').should('be.visible').within(() => {
+            cy.findByText('Mattermost: You are almost done').should('be.visible');
+        });
+
+        cy.task('getRecentEmail', {username, mailUrl}).then((response) => {
+            const messageSeparator = getEmailMessageSeparator(baseUrl);
+
+            verifyEmailInvite(response, email, messageSeparator);
+
+            const bodyText = response.data.body.text.split('\n');
+            const permalink = bodyText[6].match(reUrl)[0];
+
+            // * Check that URL in address bar does not have an `undefined` team name appended
+            cy.url().should('not.include', 'undefined');
+
+            // # Visit permalink (e.g. click on email link)
+            cy.visit(permalink);
+
+            // # Check that 'Email Verified' text should be visible, email is prefilled, and password field is focused, then login
+            cy.findByText('Email Verified').should('be.visible');
+            cy.get('#loginId').should('have.value', email);
+            cy.get('#loginPassword').should('be.focused').type(password);
```

Comment:
Could you confirm if this focus check to password is working fine on you? Thanks

Reply:
Hi Saturn, it seems to be working for me -

<img width="843" alt="Screen Shot 2020-08-27 at 11 05 51 PM" src="https://user-images.githubusercontent.com/44601888/91453249-7738e580-e8ba-11ea-8b85-e106588d0e82.png">